### PR TITLE
maya-feat: support for Maya 2023

### PR DESCRIPTION
**Pull request by sindharta**

### CODE REVIEW

File: Plugins~/Docs/en/MakeOptionalArguments.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ with commands described in the following sections.
 * Maya 2019: `-DBUILD_MAYA_2019_PLUGIN=ON`
 * Maya 2020: `-DBUILD_MAYA_2020_PLUGIN=ON`
 * Maya 2022: `-DBUILD_MAYA_2022_PLUGIN=ON`
+* Maya 2023: `-DBUILD_MAYA_2022_PLUGIN=ON`
```
Comment (seandillon92) [inline Plugins~/Docs/en/MakeOptionalArguments.md]: the flag should be -DBUILD_MAYA_**2023**_PLUGIN=ON

Comment (sindharta) [reply Plugins~/Docs/en/MakeOptionalArguments.md]: Right !

Comment (sindharta) [reply Plugins~/Docs/en/MakeOptionalArguments.md]: Fixed

**Review by seandillon92 — APPROVED** [review]
LGTM

**Pull request merged**